### DISCUSSION TITLE
Some questions about hydraulic conductivity

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2386] [Q&A]
Dear Professor, I have some questions about hydraulic conductivity to ask you.
Firstly,  the hydraulic conductivity of the throat can be obtained by the Hagen-Poiseuille equation.  How can I draw the curve of it?
Secondly, if I want to calculate the porosity distribution like the throat hydraulic conductivity distribution, how would I do it ? 
Finally,  is hydraulic conductivity equial to permeability coefficient in OpenPNM?

**Comment by ma-sadeghi** [comment]
> Firstly, the hydraulic conductivity of the throat can be obtained by the Hagen-Poiseuille equation. How can I draw the curve of it?

Do you mean how can you plot the distribution of hydraulic conductance? If so, just use `matplotlib`'s `hist`

```python
import matplotlib.pyplot as plt
plt.hist(phase["throat.hydraulic_conductance"])
```

> Secondly, if I want to calculate the porosity distribution like the throat hydraulic conductivity distribution, how would I do it ?

What exactly do you mean by "porosity distribution"?

> Finally, is hydraulic conductivity equal to permeability coefficient in OpenPNM?

I don't know the exact definition of "permeability coefficient", so I'll explain in terms of just permeability (k), as defined by:

![image](https://user-images.githubusercontent.com/14086031/164754397-7922ccec-452d-4479-9ce7-f987a01db708.png)

In `openpnm`, we refer to (k/nu/L) as "hydraulic conductance".
